src/CommonDBTM.php

```diff
@@ -6369,4 +6369,17 @@ public static function displayHelpdeskHeader(
             false
         );
     }
+
+    final public function cleanAlerts(array $types)
```

Comment:
```suggestion
    /**
     * Delete alerts of given types related to current item.
     *
     * @param array $types
     *
     * @return void
     *
     * @since 10.0.0
     */
    final public function cleanAlerts(array $types): void
```